src/cmd/run.go

```diff
@@ -63,13 +75,69 @@ var runCmd = &cobra.Command{
 			message.Fatalf(err, "Cannot unmarshal %s", config.TaskFileLocation)
 		}
 
-		if config.ListTasks {
+		ListTasks = cmd.Flag("list").Value.String() == "true"
+		ListAllTasks = cmd.Flag("list-all").Value.String() == "true"
+
+		if ListTasks || ListAllTasks {
 			rows := [][]string{
 				{"Name", "Description"},
 			}
 			for _, task := range tasksFile.Tasks {
 				rows = append(rows, []string{task.Name, task.Description})
 			}
+			// If ListAllTasks, add tasks from included files
+			if ListAllTasks {
```

Comment:
Lots of nested logic/loops going on in this block, can we break this up into functions?

Reply:
absolutely! updated in latest commit